bind_exporter.go

```diff
@@ -187,6 +187,11 @@ var (
 			"Number of failed zone transfers.",
 			nil, nil,
 		),
+		"RecursClients": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "recursive_clients_total"),
```

Comment:
If this is "current number of clients", it seems like a gauge. It should should not have `_total` in the name as that's for counters.

Reply:
Thanks @SuperQ, you're right this would be a gauge. 

What's a suitable naming for gauge based metrics? `_current` or just not including a suffix at all?

Reply:
For gauges, we have no suffix, so they end with the unit type of the metric. In this case `_clients` would be just fine.